mod-source-record-manager-server/src/main/java/org/folio/dao/JobExecutionDaoImpl.java

```diff
@@ -81,6 +88,16 @@ public class JobExecutionDaoImpl implements JobExecutionDao {
   private static final String TABLE_NAME = "job_execution";
   private static final String PROGRESS_TABLE_NAME = "job_execution_progress";
   public static final String GET_JOB_EXECUTION_HR_ID = "SELECT nextval('%s.job_execution_hr_id_sequence')";
+  public static final String ID = "id";
```

Comment:
nitpick: i think it would be nice to add a comment that says that these constants are meant to be used for substitution into a sql query that is somewhere.

Ideally, I would want something like a query builder object that will accept these values. The query builder would substitute the value and return a complete valid SQL string. That way everything is modularized and self-contained. The current approach breaks if a substitution name is changed in the other DB constants class. A developer would need to know to change both spots or any other spots used.

Reply:
Hi @okolawole-ebsco - I agree with your point. I feel we should have a generic builder for all types of queries.
We also get such libraries to use directly.
I will try to research the best fit for us.

I am thinking to go ahead with the merge and pick this change at a later stage, please share your thoughts.

Reply:
yep, that is fine. The comment should suffice for now.